Missionframework/modules/02_build/fnc/fn_build_setupPlayerActions.sqf

```diff
@@ -19,27 +19,14 @@
 
 // Actions avalible LOCALLY to player
 if(hasInterface) then {
-    ["player_fob", {
-        params ["_player", "_fob"];
-
-        private _buildActionId = _player getVariable ["KPLIB_actionId_build", nil];
-
-        // Remove redeploy action if player had one avalible
-        if (_fob isEqualTo "" || _fob isEqualTo "KPLIB_eden_startbase_marker") then {
-            if (!isNil "_buildActionId") then {
-                _player removeAction _buildActionId;
-                _player setVariable ["KPLIB_actionId_build", nil];
-            };
-        } else {
-            // If entered fob and had no action
-            if (isNil "_buildActionId") then {
-                // Add action to player
-                _buildActionId = _player addAction [localize "STR_ACTION_FOB_BUILD", {[] spawn KPLIB_fnc_build_camOpen}, nil, -802, false, true, "", "vehicle player == player", 10];
-                // Save action id so it can we removed when out of FOB
-                _player setVariable ["KPLIB_actionId_build", _buildActionId];
-            };
-        };
-    }] call KPLIB_fnc_event_addHandler
+    // Build action
```

Comment:
This action will be missing after respawn. I will fix this later.

Reply:
Added it to the review